HousingRepairsOnlineApi/Helpers/SoREngine.cs

```diff
@@ -14,9 +14,9 @@ public SoREngine(IDictionary<string, IDictionary<string, dynamic>> sorMapping, I
             this.journeyTriageOptions = journeyTriageOptions;
         }
 
-        public string MapSorCode(string location, string problem, string issue)
+        public RepairTriageDetails MapToRepairTriageDetails(string location, string problem, string issue)
```

Comment:
This whole method can be written as:
`return issue is null ? sorMapping[location][problem] : sorMapping[location][problem][issue];
`